AutoCollection/ClientRequests.ts

```diff
@@ -73,8 +73,8 @@ class AutoCollectClientRequests {
      * Tracks an outgoing request. Because it may set headers this method must be called before
      * writing content to or ending the request.
      */
-    public static trackRequest(client: Client, requestOptions: any, request: http.ClientRequest,
-            properties?: { [key: string]: string }) {
+    public static trackRequest(client: Client, requestOptions: http.RequestOptions, request: http.ClientRequest,
```

Comment:
The `requestOptions` parameter may actually be a string. (See the Node.js documentation for `http.request()`.) And that is very common usage.

So that means if it is a string you'll need to parse it in `canIncludeCorrelationHeader`, since that code currently assumes it is an object.